tests/amazon/aws/triggers/test_s3_triggers.py

```diff
@@ -11,152 +11,149 @@
 )
 
 
-def test_s3_key_trigger_serialization():
-    """
-    Asserts that the TaskStateTrigger correctly serializes its arguments
-    and classpath.
-    """
-    trigger = S3KeyTrigger(bucket_key="s3://test_bucket/file", bucket_name="test_bucket", wildcard_match=True)
-    classpath, kwargs = trigger.serialize()
-    assert classpath == "astronomer.providers.amazon.aws.triggers.s3.S3KeyTrigger"
-    assert kwargs == {
-        "bucket_name": "test_bucket",
-        "bucket_key": "s3://test_bucket/file",
-        "wildcard_match": True,
-        "aws_conn_id": "aws_default",
-        "hook_params": {},
-        "check_fn": None,
-    }
-
-
-@pytest.mark.asyncio
-@mock.patch("astronomer.providers.amazon.aws.triggers.s3.S3HookAsync.get_client_async")
-async def test_s3_key_trigger_run(mock_client):
-    """
-    Test if the task is run is in triggerr successfully.
-    """
-    mock_client.return_value.check_key.return_value = True
-    trigger = S3KeyTrigger(bucket_key="s3://test_bucket/file", bucket_name="test_bucket")
-    with mock_client:
-        task = asyncio.create_task(trigger.run().__anext__())
-        await asyncio.sleep(0.5)
+class TestS3KeyTrigger:
+    def test_s3_key_trigger_serialization(self):
+        """
+        Asserts that the TaskStateTrigger correctly serializes its arguments
+        and classpath.
+        """
+        trigger = S3KeyTrigger(
+            bucket_key="s3://test_bucket/file", bucket_name="test_bucket", wildcard_match=True
```

Comment:
`bucket_key` and `bucket_name`  value  can be made as Constant since all tests cases use it.